tasks/install.yml

```diff
@@ -0,0 +1,68 @@
+---
+- import_tasks: apt_install.yml
+  when: ansible_pkg_mgr == "apt"
+
+- name: Add gitlab user
+  user:
+      name: git
+      shell: /bin/nologin
+      comment: 'GitLab'
+      state: present
+      groups: redis
+
+- name: Install bundler gem
+  gem:
+    name: bundler
+  become: yes
+  become_user: git
+
+- import_tasks: postgres_install.yml
+
+- name: Creates redis socket directory
+  file:
+    path: "{{ redis_unixsocket | dirname }}"
+    state: directory
+    owner: redis
+    group: redis
+    mode: 0755
+
+- import_tasks: gitlab_setup.yml
+
+- name: Fetch Gitaly source with git and compile with go
+  command: 'sudo -u git -H bundle exec rake "gitlab:gitaly:install[/home/git/gitaly]" RAILS_ENV=production'
+  args:
+    chdir: "{{ gitlab_install_path | mandatory }}"
+
+- name: Restrict Gitaly socket access
+  file:
+    path: "{{ gitlab_install_path | mandatory }}/tmp/sockets/private"
```

Comment:
But what will happen when I reboot?

Reply:
is this needed? if yes use systemd tmp files